integration_tests/test_coverage.py

```diff
@@ -133,7 +133,8 @@ def test_coverage(profile, no_cleanup):
             # In the CI Profile we expect the coverage to be manually updated.
             assert False,\
                 "Coverage is increased from {} to {}. "\
-                "Please update the coverage in tests/coverage.".format(
+                "Please update the coverage in coverage_config_x86_64.json "\
+                "and coverage_config_aarch64.json.".format(
```

Comment:
Can we use `platform.machine()` instead of hardcoding the platform?

Reply:
Yes, that's better. Changed it.